TeamCharles/customer-service
# 10 build customer factory 2

## Status
**READY**

## Testing

[X ] I have added passing tests for all new functionality.

[ ] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[X ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

This PR allows for users to query database and select a customer. 

## Resolves Issue Number

#10 

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout <10-build-customer-factory-2>
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. run test
2. Within the test you will see that you are able to grab a member of the customers from the database

## Impacted Files in Application
List general components of the application that this PR will affect:

* Customer.cs
